pages/events/[id]/index.js

```diff
@@ -63,6 +60,22 @@ export default function EventDetails() {
     return <h1>No event found.</h1>;
   }
 
+  const date = new Date(event.date);
+  const options = {
+    weekday: "long",
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  };
+
+  const formattedDate = new Intl.DateTimeFormat(
+    navigator.language,
+    options
+  ).format(date);
+
```

Comment:
All this stuff belongs to a `formatDate` function as well - keep components as short as possible!

Bonus: you can reuse formatters in different components this way.

Reply:
Thanks, will refactor accordingly.